perceval/simulators/noisy_sampling_simulator.py

```diff
@@ -337,3 +343,20 @@ def sample_count(self,
         sampling = self.samples(svd, max_samples, max_shots, progress_callback)
         sampling['results'] = samples_to_sample_count(sampling['results'])
         return sampling
+
+    def log_resources(self, method: str, extra_parameters: Dict):
+        """Log resources of the noisy sampling simulator
+
+        :param method: name of the method used
+        :param extra_parameters: extra parameters to log
```

Comment:
Since the `log_resources()` method has specific extra_parameters inside each class (Processor, Simulator, etc. ), perhaps you could add the names of extra parameters in the docstring. The same comment for the other cases too.